### DISCUSSION TITLE
The look_at function should handle the case where the target is directly above on its own instead of throwing errors and requiring the user to hack around it.

### DISCUSSION THREAD

**Discussion by Melvin8D** [godotengine/godot-proposals#10200] [3D]
The Node3D "look_at" function is a simple way to have the node oriented towards a position. When the position is directly above or below the node, it throws errors, meaning the user has to handle the case where the target is directly above or below the node. This makes the "look_at" function quite annoying to use as this error will pop up when using it directly. I believe the "look at" function should handle this case on its own for convenience and usability.

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in these discussions:
* https://github.com/godotengine/godot/issues/57457
* https://github.com/godotengine/godot/issues/53793

**Reply by Melvin8D** [reply]
Ah sorry I seem to have missed these.
